pkg/apis/metrics/types.generated.go

```diff
@@ -25,9 +25,14 @@ import (
 	"errors"
 	"fmt"
 	codec1978 "github.com/ugorji/go/codec"
+	pkg2_api "k8s.io/kubernetes/pkg/api"
```

Comment:
(nit) package names should not contain underscores
https://blog.golang.org/package-names